pkg/test-infra/tests/client.go

```diff
@@ -23,42 +24,35 @@ import (
 	"k8s.io/client-go/rest"
 
 	"github.com/pingcap/tipocket/pkg/test-infra/binlog"
-	"github.com/pingcap/tipocket/pkg/test-infra/br"
 	"github.com/pingcap/tipocket/pkg/test-infra/cdc"
-	"github.com/pingcap/tipocket/pkg/test-infra/chaos"
 	"github.com/pingcap/tipocket/pkg/test-infra/fixture"
 	"github.com/pingcap/tipocket/pkg/test-infra/mysql"
 	"github.com/pingcap/tipocket/pkg/test-infra/tidb"
 
 	corev1 "k8s.io/api/core/v1"
-	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// E2eCli contains clients
-type E2eCli struct {
+// TestCli contains clients
+type TestCli struct {
 	Config *rest.Config
 	Cli    client.Client
-	Chaos  *chaos.Chaos
-	BR     *br.BrOps
 	CDC    *cdc.CdcOps
 	MySQL  *mysql.MySQLOps
 	TiDB   *tidb.TidbOps
 	Binlog *binlog.Ops
 }
 
-// NewE2eCli creates e2e client
-func NewE2eCli(conf *rest.Config) *E2eCli {
+// NewTestCli creates test client
+func NewTestCli(conf *rest.Config) *TestCli {
```

Comment:
Why do you remove Chaos and BR supports?

Reply:
Chaos was useless, BR we haven't support now.